src/cipher.rs

```diff
@@ -64,35 +62,45 @@ pub extern "C" fn rustls_cipher_get_signature_scheme_name(
 ) {
     ffi_panic_boundary_unit! {
         let write_buf: &mut [u8] = unsafe {
-            let out_n: &mut size_t = match out_n.as_mut() {
-                Some(out_n) => out_n,
-                None => return,
-            };
-            *out_n = 0;
             if buf.is_null() {
                 return;
             }
             slice::from_raw_parts_mut(buf as *mut u8, len as usize)
         };
```

Comment:
I think this is the pattern we were talking about in Slack earlier. How about:
```
if buf.is_null() {
    return;
}
let write_buf: &mut [u8] = unsafe { slice::from_raw_parts_mut(buf as *mut u8, len as usize) };
```

Reply:
(Is the Slack instance accessible to me? Might be useful...)

Reply:
Yep! Josh sent you an invite. Let me know if you didn't receive it.